ZenPacks/zenoss/ZenPackLib/tests/ZPLTestHarness.py

```diff
@@ -1,17 +1,27 @@
 import inspect
```

Comment:
ZPLTestHarness.py|1 col 1 error| 'inspect' imported but unused [F401]
ZPLTestHarness.py|165 col 9 error| local variable 'bases_from' is assigned to but never used [F841]
ZPLTestHarness.py|166 col 9 error| local variable 'bases_to' is assigned to but never used [F841]